sequelize/index.d.ts

```diff
@@ -3319,38 +3319,25 @@ declare namespace sequelize {
     /**
      * Options for Model.findOrInitialize method
      */
-    interface FindOrInitializeOptions<TAttributes> extends LoggingOptions {
-
-        /**
-         * A hash of search attributes.
-         */
-        where: string | WhereOptions;
+    interface FindOrInitializeOptions<TAttributes> extends FindOptions {
 
         /**
          * Default values to use if building a new instance
          */
         defaults?: TAttributes;
 
-        /**
-         * Transaction to run query under
-         */
-        transaction?: Transaction;
```

Comment:
If you look at line 979 of sequelize-tests.ts you can see that usage of `transaction` is covered there.